core/tests/protractor_utils/workflow.js

```diff
@@ -57,13 +58,16 @@ var checkForAddTitleWarning = async function() {
 
 // Trigger onblur event for title.
 var triggerTitleOnBlurEvent = async function() {
-  await element(by.css('.protractor-test-exploration-title-input')).click();
-  await element(by.css('.protractor-test-exploration-objective-input')).click();
+  var tstExpTitI = element(by.css('.protractor-test-exploration-title-input'));
+  await action.click('Test Exploration Title Input', tstExpTitI);
+  var tstExOI = element(by.css('.protractor-test-exploration-objective-input'));
+  await action.click('Test Exploration Objective Input', tstExOI);
```

Comment:
The variable names `tstExpTitI` and `tstExOI` are hard to understand. Could you please make them more readable?

Reply:
I used variable name tstExpTitI and tstExOI because of the word limit. First, I used the name testExplorationTitleInput as a name but showed the lint error crossed 80 words limit so I tried to make it sensible that's why I used these names. Can we increase the word limit ??

Reply:
Instead of using short variable names, you can break the line like this:

```js
var someLongButDescriptiveVariableName = element(by.css(
  '.protractor-test-some-html-class'));
```

The limit you were seeing is a character limit on line length